HACKING.md

```diff
@@ -1,288 +1,169 @@
+---------------------------------------
 
-echo "shutting down redis"
-pkill -f redis
+This section assumes that you will be using the pre-build RDO image with an IP address of 172.24.4.100.  If you are using a different image, you may need to adjust the network configuration to match your VM definition. 
 
+In order to operate with the downloaded RDO image, you may need to make a change to the definition of the vboxnet0 host-only network.  Open the VirtualBox app and navigate to the  "Host-only Networks" panel of the "Network" section of the "Preferences" menu item.
 
-echo "shutting down elasticsearch"
-pkill -f elasticsearch
+![enter image description here](https://lh3.googleusercontent.com/gV6Kh5tnOw1LKFReNVfyxkDp7uwvuG3RgFWn9fqLey8=w934-h634-no)
 
+Edit the vboxnet0 entry and set the IP address to 172.24.4.1 and the netmask to 255.255.255.0.
 
-echo "shutting down postgres"
-pkill -f postgres
-```
+![enter image description here](https://lh3.googleusercontent.com/fDkZKCZbOS4XIfB2UHktErbpVRjPRf55Li-UmLj5WP4=w973-h634-no)
 
-Activating and deactivating the environment can be done with the following commands:
+Set Up an OpenStack VM
+--------------------------------
+For convenience, you can download a VM image **[TODO add link when available]** with a Kilo version of [RDO](https://www.rdoproject.org/Main_Page), preconfigured for use with Goldstone Server.  Once downloaded, import the VM into VirtualBox.
 
-```bash
-$ workon goldstone-server
-$ deactivate
-```
+If you prefer to configure your own OpenStack, you will need to follow the instructions for configuring OpenStack hosts in the [INSTALL](http://goldstone-server.readthedocs.org/en/latest/INSTALL/) guide.  You should also update your `postactivate` script to use proper values for the `OS_*` settings.
 
-### Install some packages
+Configure boot2docker VM Port Forwarding
+----------------------------------------------------
+This section assumes that you are using the pre-build RDO image with an IP address of 172.24.4.100.  If you are using a different image, you may need to adjust the source IP address to match your VM.  
 
-Install these packages in your Goldstone virtualenv:
+Expand the "Advanced" section of the Network settings for the boot2docker-vm, and click on the "Port Forwarding" button.  Then configure forwarders as described in the table below.
+|Name|Protocol|Host IP|Host Port|Guest IP|Guest Port|
+|----|--------|-------|---------|--------|----------|
+|es_9200_RDO|TCP|172.24.4.1|9200||9200|
+|es_9200_local|TCP|127.0.0.1|9200||9200|
+|es_9300_RDO|TCP|172.24.4.1|9300||9300|
+|es_9300_local|TCP|127.0.0.1|9300||9300|
+|logstash_syslog_RDO|TCP|172.24.4.1|5514||5514|
+|logstash_syslog_local|TCP|127.0.0.1|5514||5514|
+|logstash_metrics_RDO|UDP|172.24.4.1|5516||5516|
+|logstash_metrics_local|UDP|127.0.0.1|5516||5516|
+|postgres_RDO|TCP|172.24.4.1|5432||5432|
+|postgres_local|TCP|127.0.0.1|5432||5432|
+|redis_local|TCP|127.0.0.1|6379||6379|
+|ssh|TCP|127.0.0.1|2022||22|
```

Comment:
This table is broken. 

![screen shot 2015-06-06 at 11 45 32 am](https://cloud.githubusercontent.com/assets/231148/8020946/91c3545e-0c41-11e5-9f40-15926189d9ce.png)